web/components/geo/map/FcMapPopup.vue

```diff
@@ -81,6 +90,9 @@ export default {
     featureSelectable() {
       return SELECTABLE_LAYERS.includes(this.feature.layer.id);
     },
+    isStudyRequest() {
+      return !(this.feature.properties.studyRequests === undefined);
```

Comment:
no biggie, i'd use `!==`

Reply:
I'm now left baffled as to why I didn't do that 😅